frontend/packages/noobaa-storage-plugin/src/components/data-consumption-card/data-consumption-card-dropdown.tsx

```diff
@@ -48,7 +48,7 @@ export const DataConsumptionDropdown: React.FC<DataConsumptionDropdownProps> = (
       id="egress"
       key="egress"
       component="button"
-      isDisabled={type === typesDropdown.accounts}
+      className={`${type === typesDropdown.accounts? 'nb-data-consumption-card__dropdown-item-egress' : ''}`}
```

Comment:
you can just not add this item to the array if `type === typesDropdown.accounts`